Instruments/source/BTagEffEstimator.cxx

```diff
@@ -21,9 +21,10 @@ struct Arguments { // list of all program arguments
     REQ_ARG(std::string, output_file);
     OPT_ARG(std::string, apply_pu_id_cut,"no");
     OPT_ARG(unsigned, n_threads, 1);
-    OPT_ARG(std::string, deep_csv,"yes");
-    REQ_ARG(std::vector<std::string>, input_file);
+    OPT_ARG(std::string, csv_type,"deepCSV");
```

Comment:
For csv_type and period, you can use JetOrdering and Period enums here instead of std::string.